sipa/model/hss/user.py

```diff
@@ -269,17 +270,33 @@ def userdb(self):
     def has_connection(self):
         return self._pg_account.properties.active
 
-    @active_prop
-    @money
-    def finance_balance(self):
-        return self._pg_account.finance_balance
+    @property
+    def finance_information(self):
+        return FinanceInformation.from_pg_account(self._pg_account)
+
+
+class FinanceInformation(BaseFinanceInformation):
```

Comment:
@sebschrader I would like your opinion on the design here, since it appears to me that i repeat myself a lot passing `balance` via `__init__` to store it in `_raw_balance` to be read out by `_balance` to be used by `balance`. Is there a better way? How useful or even harmful is the approach using an ABC?